src/TimeGrid.jsx

```diff
@@ -74,6 +74,17 @@ export default class TimeGrid extends Component {
       this.measureGutter()
     }
     this.applyScroll();
+
+    this.positionTimeIndicator();
+
+    // Update the position of the time indicator every minute
+    this._timeIndicatorInterval = window.setInterval(() => {
```

Comment:
I'm not to concerned about an interval that is triggered every second. Using recursive `setTimeout()` we might exceed the stack trace of the browser after having the calendar open for ~six hours. ;-)
Anyway, I did change to code to work with `setTimeout()`.

I also noticed that I forgot to check for the min time frame (`this.props.min`) and added this check.

The indicator is now rendered **behind** the event.